### ISSUE TITLE
ipython3 Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by biycyr** [thtrieu/darkflow#189]
when I tried to use the script **flow** in ipython3, I encounter Segmentation fault (core dumped) when I try to auto-complete(pressing Tab). 
`In [1]: from darkflow.net.build import TFNet
   ...: from tensorflow import flags
   ...: import os
   ...:
   ...: flags.DEFINE_string("test", "./test/", "path to testing directory")
   ...: flags.DEFINE_string("binary", "./bin/", "path to .weights directory")
   ...: flags.DEFINE_string("config", "./cfg/", "path to .cfg directory")
   ...: flags.DEFINE_string("dataset", "/home/ycliu/data/VOCdevkit/VOC2012/JPEGImages/", "path to dataset directory")
   ...: flags.DEFINE_string("backup", "./ckpt/", "path to backup folder")
   ...: flags.DEFINE_string("summary", "./summary/", "path to TensorBoard summaries directory")
   ...: flags.DEFINE_string("annotation", "/home/ycliu/data/VOCdevkit/VOC2007/Annotations/", "path to annotation directory")
   ...: flags.DEFINE_float("threshold", 0.1, "detection threshold")
   ...: flags.DEFINE_string("model", "cfg/yolo-voc.cfg", "configuration of choice")
   ...: flags.DEFINE_string("trainer", "rmsprop", "training algorithm")
   ...: flags.DEFINE_float("momentum", 0.0, "applicable for rmsprop and momentum optimizers")
   ...: flags.DEFINE_boolean("verbalise", True, "say out loud while building graph")
   ...: flags.DEFINE_boolean("train", False, "train the whole net")
   ...: flags.DEFINE_string("load", "bin/darknet19_448.weight", "how to initialize the net? Either from .weights or a checkpoint, or even from scratch")
   ...: flags.DEFINE_boolean("savepb", False, "save net and weight to a .pb file")
   ...: flags.DEFINE_float("gpu", 0.0, "how much gpu (from 0.0 to 1.0)")
   ...: flags.DEFINE_float("lr", 1e-5, "learning rate")
   ...: flags.DEFINE_integer("keep",20,"Number of most recent training results to save")
   ...: flags.DEFINE_integer("batch", 16, "batch size")
   ...: flags.DEFINE_integer("epoch", 1000, "number of epoch")
   ...: flags.DEFINE_integer("save", 2000, "save checkpoint every ? training examples")
   ...: flags.DEFINE_string("demo", '', "demo on webcam")
   ...: flags.DEFINE_boolean("profile", False, "profile")
   ...: flags.DEFINE_boolean("json", False, "Outputs bounding box information in json format.")
   ...: flags.DEFINE_boolean("saveVideo", False, "Records video from input video or camera")
   ...: FLAGS = flags.FLAGS
   ...:
   ...: # make sure all necessary dirs exist
   ...: def get_dir(dirs):
   ...: ^Ifor d in dirs:
   ...: ^I^Ithis = os.path.abspath(os.path.join(os.path.curdir, d))
   ...: ^I^Iif not os.path.exists(this): os.makedirs(this)
   ...: get_dir([FLAGS.test, FLAGS.binary, FLAGS.backup, os.path.join(FLAGS.test,'out'), FLAGS.summary])
   ...:
   ...: # fix FLAGS.load to appropriate type
   ...: try: FLAGS.load = int(FLAGS.load)
   ...: except: pass
   ...:
   ...:
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcublas.so.8.0 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcudnn.so.5 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcufft.so.8.0 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcuda.so.1 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcurand.so.8.0 locally

In [2]: FLASegmentation fault (core dumped)
`

However, if I use python2, this will not happen.
anyone has similar problems?
